src/Umbraco.Web.UI.Client/src/common/directives/components/application/umbappheader.directive.js

```diff
@@ -84,6 +84,35 @@
                 overlayService.open(dialog);
             };
 
+            scope.logoModal = {
+                show: false,
+                text: "",
+                timer: null
+            };
+            scope.showLogoModal = function() {
+                $timeout.cancel(scope.logoModal.timer);
+                scope.logoModal.show = true;
+                scope.logoModal.text = "version "+Umbraco.Sys.ServerVariables.application.version;
+                $timeout(function () {
+                    const anchorLink = element[0].querySelector('.umb-app-header__logo-modal');
+                    if(anchorLink) {
+                        anchorLink.focus();
+                    }
+                });
+            }
+            scope.keepLogoModal = function() {
+                $timeout.cancel(scope.logoModal.timer);
+            }
+            scope.hideLogoModal = function() {
+                $timeout.cancel(scope.logoModal.timer);
+                scope.logoModal.timer = $timeout(function () {
+                    scope.logoModal.show = false;
+                }, 100);
+            }
+            scope.stopClickEvent = function($event) {
+                $event.stopPropagation();
+            }
```

Comment:
Ideally these named funcitons should be closed with `;` ... but I think most minifiers and modern browsers can handle this.